### PR TITLE
Dispatch an event when the refresh token entity is created but not persisted yet

### PR DESCRIPTION
**Pull request by SherinBloemendaal**
https://github.com/markitosgv/JWTRefreshTokenBundle/issues/106

### CODE REVIEW

**Comment by luffy1727** [comment]
why did you close this pull request?

**Comment by SherinBloemendaal** [comment]
@luffy1727 because some tests fail and i don't know how to fix them. Besides the library owner does not reply or anything like that so i consider it as denied.

**Comment by fd6130** [comment]
> @luffy1727 because some tests fail and i don't know how to fix them. Besides the library owner does not reply or anything like that so i consider it as denied.

Maybe you can ping someone who can make decision for this PR?

**Comment by SherinBloemendaal** [comment]
> > @luffy1727 because some tests fail and i don't know how to fix them. Besides the library owner does not reply or anything like that so i consider it as denied.
> 
> Maybe you can ping someone who can make decision for this PR?

Okay, i reopend the PR and i will have a deeper look at the tests today if i have time. When the tests succeed i will ping the owner.

**Comment by SherinBloemendaal** [comment]
The tests are fixed. @markitosgv what do you say?

File: Event/RefreshTokenEvent.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 use Gesdinet\JWTRefreshTokenBundle\Model\RefreshTokenInterface;
 use Symfony\Component\Security\Guard\Token\PostAuthenticationGuardToken;
 
-class RefreshEvent extends Event
+class RefreshTokenEvent extends Event
```
Comment (maxhelias) [inline Event/RefreshTokenEvent.php:17]: It's backward compatibility. You should make a new class and extend it with `RefreshEvent` and trigger a deprecation on the old one (in phpdoc or construct i don't know how it's managed for this project).

Comment (SherinBloemendaal) [reply Event/RefreshTokenEvent.php:17]: Okay thanks!

**Pull request closed without merging**